### ISSUE TITLE
When notification is in "show less" state, controls are not keyboard-accessible

### ISSUE THREAD

**Issue by cait1inhennessy** [Microsoft/vscode#57985]
This accessibility bug was filed on the Live Share team; however, it is really an editor issue.  I've pasted the bug description below, editing it slightly to remove irrelevant Live Share-specific content.

Repro Steps:
1) Show any notification with the vscode.window.showNotificationMessage API
2) Check whether all the controls in the notification are accessible through keyboard when it is in "Show less(collapsed)" state.

Actual Result:
When the notification is in "Show less" state, focus is moving out of the notification without going to other controls of the notification

Expected Result:
All the controls in the notification pop-up should be accessible through keyboard

User Impact: 
Keyboard only users will not be able to access the controls present in the notification

MAS Reference:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

OS: Windows 10 Enterprise 17134.228

**Comment by bpasero** [comment]
@cait1inhennessy I cannot reproduce:

![before](https://user-images.githubusercontent.com/900690/45140154-3e297700-b1b2-11e8-9201-58a1a5223920.gif)

Let me know what I am doing different.

**Comment by bpasero** [comment]
@cait1inhennessy I cannot reproduce:

![before](https://user-images.githubusercontent.com/900690/45199449-07179c00-b26c-11e8-8738-44798381e099.gif)

**Comment by cait1inhennessy** [comment]
@bpasero I think the key is you have to click on the notification with the mouse in order to make the icons accessible in 'show less' mode.  (At least, that's what I saw when I was testing it myself today.)  For accessibility purposes, this needs to work without the mouse click.

**Comment by bpasero** [comment]
@cait1inhennessy for that reason we have a command "Show Notifications" to move keyboard focus into the notifications and then you can use tabbing to get there.

**Comment by cait1inhennessy** [comment]
@bpasero Could you provide more details about this command? I can't find it in the palette:

![capture](https://user-images.githubusercontent.com/16658549/45564852-93930300-b806-11e8-84a8-a4e4f6c7cb19.PNG)

**Comment by cait1inhennessy** [comment]
@bpasero Never mind! Realized the command is only available when the notifications center is collapsed. Confirmed the tab behavior works as you described.
